website_upload_video/__openerp__.py

```diff
@@ -8,6 +9,7 @@
     'category': 'Website',
     'version': '8.0.1.0.0',
     'author': 'ONESTEiN BV,Odoo Community Association (OCA)',
+              'Techspawn Solutions'
```

Comment:
With your little change, you deserve to be on the contributors section of the README, but not in the co-author list I think.